src/strategies/balance-of-subgraph/index.ts

```diff
@@ -0,0 +1,55 @@
+import { getAddress } from '@ethersproject/address';
+import { subgraphRequest } from '../../utils';
+
+const SUBGRAPH_URL = {
+  '1': 'https://api.thegraph.com/subgraphs/name/dinngodev/furucombo-tokenomics-mainnet',
+  '137':
+    'https://api.thegraph.com/subgraphs/name/dinngodev/furucombo-tokenomics-polygon'
+};
+
```

Comment:
Actually, the problem is with the name of the strategy @weizard It sounds like a core strategy that can be used by anyone 😄 but we hardcoded the subgraphs here, input and output fields so won't be useful for others. can find a way to pass these dynamically. or change the name to something like `furucombo-balance`